_posts/pl/2023-07-24-czy-wiesz-czym-jest-indexeddb.md

```diff
@@ -30,6 +30,28 @@ Dodatkowo IndexedDB pozwala na:
 - obsługę indeksów,
 - przechowywanie znacznie większej ilości danych niż localStorage i cookies.
 
+## Maksymalna ilość danych przechowywanych w IndexedDB
+Maksymalna ilość danych, jaką można przechowywać w IndexedDB, różni się w zależności od przeglądarki internetowej, a ograniczenia te mogą wynosić do kilku gigabajtów.
+
+### Firefox
+Firefox nakłada ograniczenia w zależności od rozmiaru profilu użytkownika na dysku:
+- 10% całkowitego rozmiaru dysku dla profilu użytkownika,
+- 10 GB dla wszystkich stron z tej samej domeny (limity grupowe),
+- Do 50% całkowitego rozmiaru dysku, z ograniczeniem do 8 TB, dla stron z przyznaną pamięcią trwałą.
+
+#### Przykład:
+Jeśli nasze urządzenie ma dysk o pojemności 1000 GB, Firefox pozwoli źródłu na przechowywanie do:
+- 10 GB danych (limit grupy dla tej samej domeny),
+- 500 GB danych (50% całkowitego rozmiaru dysku).
+
+### Chrome i przeglądarki oparte na projekcie open-source Chromium (np. Edge)
+Przeglądarki te pozwalają źródłu na przechowywanie do 60% całkowitego rozmiaru dysku.
```

Comment:
ciekawe czy jest to odporne przed jakimś złośliwym oprogramowanie :thinking: 
bo łatwo sobie wyobrazić scenariusz, że coś nam zapycha tą przestrzeń, a monitorowania tego ile zajmują nasze dane w IndexedDB nie jest takie oczywiste.
Ale moim zdaniem te informacje wystarczą, nie ma co wchodzić chyba głębiej w to.